src/Resources/contao/languages/en/modules.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of terminal42/contao-inserttags.
+ *
+ * (c) terminal42
+ *
+ * @license MIT
+ */
+
+// Back end modules
+$GLOBALS['TL_LANG']['MOD']['inserttags'] = ['Insert-Tags', 'This module allows you to create custom insert-tags and let them be replaced by some content. This can be very helpful if you have some content wich you want to have on multiple pages, like your phone number. If your phone number changes, you will need to change it once, and will never forget to update a subpage.'];
```

Comment:
```suggestion
$GLOBALS['TL_LANG']['MOD']['inserttags'] = ['Insert-Tags', 'This module allows you to create custom insert tags and let them be replaced by some content. This can be very helpful if you have some content wich you want to have on multiple pages, like your phone number. If your phone number changes, you will need to change it once, and will never forget to update a subpage.'];
```